compliance_checker/cf/cf.py

```diff
@@ -723,9 +725,9 @@ def check_ancillary_data(self, ds):
             anc_result = Result(BaseCheck.HIGH, name=('§3.4 Ancillary Variables', k))
             msgs = []
 
-            if not isinstance(anc, str):
+            if not isinstance(anc, basestring):
```

Comment:
https://github.com/DanielJMaher/compliance-checker/blob/ancillary/compliance_checker/cf/cf.py#L12-L15 defines it